modules/order/tests/src/Kernel/OrderReceiptTest.php

```diff
@@ -77,7 +77,7 @@ protected function setUp() {
     $product->addVariation($variation1)->save();
 
     $profile = Profile::create([
-      'type' => 'customer',
+      'type' => OrderType::PROFILE_COMMON,
```

Comment:
**OPTIONAL CHANGE REQUEST**

The constant is defined at the interface and it is not really tied to the `\Drupal\commerce_order\Entity\OrderType` object. It's best to use `OrderTypeInterface::PROFILE_COMMON`. The same pretty much everywhere in the code really, even though `OrderType::` works as well.